models/comments.models.js

```diff
@@ -12,4 +12,14 @@ exports.selectCommentsByArticleId = (article_id) => {
         }
         return rows;
     })
-}
\ No newline at end of file
+}
+
+exports.insertCommentsByArticleId = (article_id, newComment) => {
+        return db.query(`
+    INSERT INTO comments
+    (article_id, author, body)
+    VALUES
+    ($1, $2, $3)
+    RETURNING *
+    `, [article_id, newComment.username, newComment.body])
+    }
```

Comment:
Ideally we should be chaining a .then onto this and returning rows[0] from the model, serving back to the controller the object to send back to the user. This is generally how you will find the conventional relationship between the controller and the model.